### ISSUE TITLE
Extension inactive if the Slidev project is in a sub-folder

### ISSUE THREAD

**Issue by davelsio** [slidevjs/slidev-vscode#10]
## Describe

The Slidev folder must be opened in VSCode as the root workspace folder. If the project is in a sub-folder, the extension is not activated, resulting in:

- A missing extension side panel.
- All commands returning an error. See #7 for an image (same error for every command).

## Reproduce

- With the extension installed, open a folder that contains a sub-folder with a Slidev project.
- Verify that the Slidev side panel is missing.
- Attempt to run any Slidev command.

## Notes

This might be the expected behavior. If so, perhaps there is a way to warn the user about this?

**Comment by davelsio** [comment]
Reconverted the issue to better describe the actual problem. Reopening.

**Comment by antfu** [comment]
Have `@slidev/cli` in your root `pakcage.json` and it will work.

**Comment by microboym** [comment]
<img width="940" alt="Screen Shot 2022-07-14 at 11 47 38" src="https://user-images.githubusercontent.com/48620758/178893929-eeda4af9-d814-4ce6-8acd-a19fbc97124f.png">
I have installed slidev globally. I created `slides.md` in the root folder of vscode and ran `slidev` in the terminal, which successfully started the presentation that I could open in my browser. I can not find any `.json` file in my folder. (is the spelling correct in `pakcage.json`?)

@antfu 
> Have `@slidev/cli` in your root `pakcage.json` and it will work.
